### PR TITLE
More links to help

### PR DESCRIPTION
**Pull request by Maximus619**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -118,3 +119,10 @@ I have been collecting many links throughout months and I wish to share some of
 - [Can I Use](https://caniuse.com/)
 - [Carbon](https://carbon.now.sh/)
 - [Wappalyzer](https://www.wappalyzser.com/)
+- [NODEJS](https://nodejs.org/en/)
```
Comment (jatin2003) [inline README.md:122]: It's not a quality resource 😔

File: README.md [hunk 2]
```diff
@@ -118,3 +119,10 @@ I have been collecting many links throughout months and I wish to share some of
...
+- [CHROME WEB DEVLOPER TOOLS](https://chrome.google.com/webstore/category/ext/11-web-development)
```
Comment (jatin2003) [inline README.md:123]: It doesn't point to any single chrome extension for developers 😅

File: README.md (hunk 1 above)
Comment (Maximus619) [reply README.md:122]: I thought it would be.I will get more good resources soon

**Comment by jatin2003** [comment]
I'm closing this pull request @Maximus619 😥 Thanks for your effort 😀

**Pull request closed without merging**